services/app-api/src/postgres/contractAndRates/updateDraftContractWithRates.ts

```diff
@@ -123,6 +142,14 @@ async function updateDraftContractWithRates(
 
     try {
         return await client.$transaction(async (tx) => {
+            const foundBeforeContract = await findContractWithHistory(
+                tx,
+                contractID
+            )
```

Comment:
Curious if this is just to make sure the contract exists or was the another purpose? The resolver queries for the contractID before calling this handler.

Reply:
great catch, this was for some debugging. It's gone now.